assimilation_code/modules/utilities/netcdf_utilities_mod.f90

```diff
@@ -63,7 +63,7 @@ module netcdf_utilities_mod
           nc_begin_define_mode,           &
           nc_end_define_mode,             &
           nc_synchronize_file,            &
-          NF90_MAX_NAME, NF90_MAX_VAR_DIMS
+          NF90_MAX_NAME, NF90_MAX_VAR_DIMS, NF90_FILL_REAL
```

Comment:
i'm not sure when the NF90 constants got added to this fie, but the intent was never to replicate the actual netcdf utilities mod.  adding more constants to this is just going down the wrong path.  these functions were intended to be more user-friendly ways to access netcdf without having to get into the weeds of the netcdf interface.  if you need these constants in your code, 'use netcdf' or make a cover function that has the name of what you're doing and uses this constant inside the function.  e.g. from the name i'm guessing it fills an array.  make a 
nc_fill_array_blank(array) that doesn't require netcdf constants in the calling code.  

i know this isn't an important fix, but it's a long-term bad idea.  maybe after the crush of getting this working is done, someone can go back and make this better?

Reply:
Yes, I didn't do much work to figure out the existing strategy for using netcdf parameters.
The template I was working from used some, so I assumed it was not terrible to do that.

Reply:
which is the only reason i brought it up.  once the camel's nose is in the tent...   
it's *not* important for this pull request and a later change to your code will be easy.  i will open an issue on this because i'm not requesting a change to this pull request, so this is the wrong place to address this.